controllers/toolchainstatus/toolchainstatus_controller.go

```diff
@@ -593,8 +593,18 @@ func ExtractStatusMetadata(instance *toolchainv1alpha1.ToolchainStatus) []*Compo
 		cond, found = condition.FindConditionByType(instance.Status.HostOperator.Conditions, toolchainv1alpha1.ConditionReady)
 		if found && cond.Status != corev1.ConditionTrue {
 			result = append(result, &ComponentNotReadyStatus{
-				ComponentType: "",
-				ComponentName: "Host Operator",
+				ComponentType: "Host Operator",
```

Comment:
shouldn't it be 
```
ComponentName: "Host Operator",
ComponentType: "Deployment",
```
instead?

Reply:
TBH I just followed the already exiting convention where component type was either: "Host Operator", "Member" or "Registration Service". I can Invert those , but if we do this we need to invert also the other existing occurrences not only the ones I've added/changed right ?

Reply:
inverted , please see: https://github.com/codeready-toolchain/host-operator/pull/836/commits/54d573742cad4df097d7652a5ae06a356182acf0